share/completions/fastboot.fish

```diff
@@ -26,7 +26,7 @@ complete -c fastboot -s v -l verbose -d 'Verbose output'
 complete -c fastboot -l version -d 'Display version'
 
 complete -n "not __fish_seen_subcommand_from $commands" -c fastboot -s w -d 'Wipe userdata'
-complete -n "not __fish_seen_subcommand_from $commands" -c fastboot -s s -d 'Specify a device'
+complete -n "not __fish_seen_subcommand_from $commands" -c fastboot -o s -x -a "(fastboot devices)" -d 'Specify a device'
```

Comment:
Why `-o`? That makes it an "old-style" option, which for things with a single character means they can't be grouped.

Can you do `fastboot -ws SOMETHING`?

Reply:
Yes, `fastboot -ws SOMETHING` works. Changed back to `-s`.